package.json

```diff
@@ -65,21 +67,23 @@
     "babel-preset-es2015": "^6.22.0",
     "babel-preset-react": "^6.23.0",
     "babel-preset-stage-1": "^6.22.0",
+    "babel-plugin-transform-builtin-extend": "^1.1.0",
     "babel-runtime": "^6.23.0",
     "cross-env": "^5.0.2",
     "css-loader": "^0.28.4",
+    "deep-equal": "^1.0.1",
     "enzyme": "^2.4.1",
     "eslint": "^3.7.1",
     "eslint-config-netlify": "github:netlify/eslint-config-netlify",
     "eslint-import-resolver-webpack": "^0.8.3",
     "exports-loader": "^0.6.4",
     "extract-text-webpack-plugin": "^2.1.2",
     "file-loader": "^0.11.2",
-    "gotrue-js": "^0.9.3",
     "identity-obj-proxy": "^3.0.0",
     "imports-loader": "^0.7.1",
     "jest": "^20.0.4",
     "jest-cli": "^20.0.4",
+    "json-loader": "^0.5.4",
```

Comment:
Looks like this isn't being used at all, no?

Reply:
Done.